submissions/ORybka/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,67 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/ORybka/a-tiny-JS-world
+   Web app: https://orybka.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hey! What\'s up?',
+   friends: ['Mary', 'Marley', 'Tisha']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Mary',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hey there!',
+   friends: ['John', 'Marley', 'Tisha']
+};
+
+const dog = {
+   species: 'dog',
+   name: 'Marley',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!',
+   friends: ['John', 'Mary', 'Selina']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Tisha',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'meow-purr!',
+   friends: []
+};
+
+const catWoman = {
+   species: 'catWoman',
+   name: 'Selina',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: ['Tisha']
+};
+
+// ======== OUTPUT ========
+
+const objects = [man, woman, dog, cat, catWoman];
+
+objects.forEach(el => {
+   const objFriends = el.friends.join(', ')
+   const objKeys = [el.species, `<strong>${el.name}</strong>`, el.gender, el.legs, el.hands, `<em>${el.saying}</em>`, objFriends];
```

Comment:
Confusing variable naming. Please fix.

Reply:
I always have this problem! I think I will ask about proper naming during next AMA. For me, it's like being a poet, very difficult topic.
Thank you! I will try to do it in a proper way

Reply:
Try reverse-checking.
Variable name is `objKeys`. In plain language this means 'object keys'. Does this variable contain object keys or something else?

Reply:
Consider googling "how to name a variable properly". It may bring many materials that would help you to master this wizardry. [Example](https://hackernoon.com/the-art-of-naming-variables-52f44de00aad)

Reply:
... and semicolones. Lines 64 & 66. Consider using eslint and prettier to do the dirty job for you.

Reply:
Thank you for the material! Will also learn more!
I hope I rewrite the name right:) 
That was not easy but I succeeded with eslint, so now my code will be cleaner. Thank you! :)